pkg/domain/transformer.go

```diff
@@ -0,0 +1,47 @@
+package domain
+
+import (
+	logger "github.com/sirupsen/logrus"
+	"strconv"
+	"strings"
+)
+
+func getTransMap(s string) (map[interface{}]interface{}, error) {
```

Comment:
the return should be a float[float] as you are parsing to a float. this well prevent a need for further casting in follow up code